app/controllers/single_use_links_viewer_controller.rb

```diff
@@ -17,53 +14,7 @@ def initialize(user, single_use_link)
       can :read, ActiveFedora::Base do |obj|
         single_use_link.valid? && single_use_link.itemId == obj.id && single_use_link.destroy!
       end if single_use_link
-    end
```

Comment:
@weiweishi Just curious, why did you leave the Ability here instead of in the behavior?